dataplicity/device_meta.py

```diff
@@ -36,3 +38,17 @@ def get_os_version():
     # Linux is a fair assumption for now
     distro = " ".join(platform.linux_distribution()).strip()
     return distro
+
+def get_ip_address_list():
+    # Get the ip addresses from all the interfaces
+    try:
+        interfaces = get_all_interfaces()
+    except:
+        '''
```

Comment:
You're forgiven my son. As long as you log it, replace the string with a comment, and say five Hail Mary's.